101-sqlazure-ragr/azuredeploy.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "primaryDBResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the source Azure SQL Database Resource Group."
+      }
+    },
+    "primarySQLServerName": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the name of the source Azure SQL Server"
+      }
+    },
+    "primarySQLDBName": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the name of the source Azure SQL Database"
+      }
+    },
+    "copyRegion": {
+      "type": "string",
+      "defaultValue": "East US",
+      "allowedValues": [
+        "Japan East",
+        "East US 2",
+        "Southeast Asia",
+        "South Central US",
+        "West Europe"
+      ],
+      "metadata": {
+        "description": "Location of the secondary Azure SQL Read-access online copy"
+      }
+    },
+    "copyServerName":{
+      "type": "string",
+      "defaultValue":"copySQLServer",
+      "metadata": {
+        "description":"Name of the new SQL Server (Copy)"
+      }
+    },
+    "copyDBName":{
+      "type": "string",
+      "defaultValue":"copySQLDB",
+      "metadata": {
+        "description":"Name of the new SQL Server Database (Copy)"
+      }
+    },
+    "sqlServerAdminLogin":{
+      "type": "string",
+      "metadata": {
+        "description":"Username for the SQL Admin Login"
+      }
+    },
+    "sqlServerAdminPassword":{
+      "type": "securestring",
+      "metadata": {
+        "description":"Password for the SQL Admin Login"
+      }
+    },
+    "sqlEdition":{
+      "type": "string",
+      "defaultValue":"Basic",
+      "metadata": {
+        "description":"Edition of the New SQL Server Database (Copy)"
+      }
+    },
+    "sqlRequestedServiceObjectiveName":{
+      "type": "string",
+      "defaultValue":"Basic",
+      "metadata": {
+        "description":"Tier of the New SQL Server Database (Copy)"
+      }
+    }
+  },
+  "variables": {
+    "apiVersion": "2014-04-01-preview",
+    "sourceSQLServerId":"[resourceId(parameters('primaryDBResourceGroup'), 'Microsoft.Sql/servers', parameters('primarySQLServerName'))]",
```

Comment:
use consistent naming in params & vars (i.e. use source or primary but not both)